pkg/status/status.go

```diff
@@ -151,3 +151,20 @@ func MessageFromError(err error) string {
 	}
 	return unwErr.Error()
 }
+
+func resetIncomparableFields(status *nropv1.NUMAResourcesOperatorStatus) {
+	resetIncomparableConditionFields(&status.Conditions)
+}
+
+func resetIncomparableConditionFields(conditions *[]metav1.Condition) {
+	for idx, _ := range *conditions {
+		(*conditions)[idx].LastTransitionTime = metav1.Time{}
+		(*conditions)[idx].ObservedGeneration = 0
+	}
+}
+
+func getShortcut(conditions []metav1.Condition) []metav1.Condition {
```

Comment:
why is this a shortcut?

Reply:
Thanks for the comment. Bad name, intended as in get a copy not really shortcut, will change that to clone instead does that sound better?

Reply:
just `DeepCopy()` maybe?

Reply:
changed to clone but DeepCopy is fine and goes well with the convention. But why do we need to export it?

Reply:
no need, private `clone` is fine